Sources/Networking/Operations/PostReceiptDataOperation.swift

```diff
@@ -41,7 +41,7 @@ final class PostReceiptDataOperation: CacheableNetworkOperation {
     ) -> CacheableNetworkOperationFactory<PostReceiptDataOperation> {
         let cacheKey =
         """
-        \(configuration.appUserID)-\(postData.isRestore)-\(postData.receiptData.asFetchToken)
+        \(configuration.appUserID)-\(postData.isRestore)-\(postData.receiptData.hashString)
```

Comment:
✅ This is the core of this change

Reply:
this is a fantastic solution :chefskiss:
let's add a comment here explaining that we're explicitly using the `hashString` since the `fetchToken` could be huge if there are a lot of transactions in the receipt

Reply:
Good idea, I added this comment:
```swift
/// Cache key comprises of the following:
/// - `appUserID`
/// - `isRestore`
/// - Receipt (`hashString` instead of `fetchToken` to avoid big receipts leading to a huge cache key)
/// - `ProductRequestData.cacheKey`
/// - `presentedOfferingIdentifier`
/// - `observerMode`
/// - `subscriberAttributesByKey`
```